backend/analytics_server/mhq/service/code/pr_analytics.py

```diff
@@ -10,6 +10,9 @@ def __init__(self, code_repo_service: CodeRepoService):
 
     def get_prs_by_ids(self, pr_ids: List[str]) -> List[PullRequest]:
         return self.code_repo_service.get_prs_by_ids(pr_ids)
+    
+    def get_prs_not_reviewed_merged(self,team_id):
```

Comment:
```suggestion
    def get_prs_merged_without_review(self,team_id: str, interval: Interval):
```

Reply:
We need to fetch these PRs in an interval otherwise we would fetch all prs in a DB, this will keep growing as the server syncs more data.

Reply:
@samad-yar-khan how should i include the interval here ? any example in the codebase i could follow